bool trigger
float32 sleep
geometry_msgs/Pose pose
float32 pos_aprox
float32 ori_aprox